packages/lambda/src/helpers/donation.js

```diff
@@ -24,6 +24,41 @@ const SettingHelper = require('./setting');
 exports.TYPE_BULK = 'BULK';
 exports.TYPE_SINGLE = 'SINGLE';
 
+exports.last4Fields = [
```

Comment:
The other report had better labels for the fields, not a huge deal.

Reply:
Agreed. I just copied what Andrew had. didn't know if it was part of another process he had